### ISSUE TITLE
After reflash no  AP

### ISSUE THREAD

**Issue by Italque** [martin-ger/esp_wifi_repeater#121]
Hi after a week I decided to reflash my ESP8266. This time at the first boot there isn't a "MyAP". I tried many times to erase flash (also with esptool) end write it again. Every time there isn't an AP. Obviously the ESP8266 is not burned. What could I do?  Thanks in advance.

**Comment by martin-ger** [comment]
Did you try DIO mode?

**Comment by Italque** [comment]
Thanks for reply! Yes I tried DIO mode as said in the Readme.

Inviato da iPhone

Il giorno 05 gen 2018, alle ore 17:36, martin-ger <notifications@github.com<mailto:notifications@github.com>> ha scritto:


Did you try DIO mode?

**Comment by martin-ger** [comment]
Pleased check the output on the serial port.


Am 5. Januar 2018 9:09:18 nachm. schrieb Italque <notifications@github.com>:

**Comment by Italque** [comment]
Is there a way to check the serial output inside the Espressif flash download Tool?
Could you point me the right way to do this?

Il giorno 05 gen 2018, alle ore 21:22, martin-ger <notifications@github.com<mailto:notifications@github.com>> ha scritto:

Pleased check the output on the serial port.


Am 5. Januar 2018 9:09:18 nachm. schrieb Italque <notifications@github.com<mailto:notifications@github.com>>:

**Comment by Italque** [comment]
I used putty for the serial port at different baud values but the result is not understandable. I also uploaded an example sketch for a Soft AP and it works.



Il giorno 05 gen 2018, alle ore 21:22, martin-ger <notifications@github.com<mailto:notifications@github.com>> ha scritto:

Pleased check the output on the serial port.


Am 5. Januar 2018 9:09:18 nachm. schrieb Italque <notifications@github.com<mailto:notifications@github.com>>:

**Comment by martin-ger** [comment]
Did you try this: 
Sometimes, especially on old ESP-01s, there is a wrong or non-matching version of "esp_init_data_default.bin" in the flash. If the firmware files from above flash correctly but after reboot you see only garbage on the serial and/or the LED on GPIO2 is flashing rapidly, try to re-initialize this sector: download https://github.com/espressif/ESP8266_NONOS_SDK/raw/master/bin/esp_init_data_default_v08.bin and flash it to 0x7c000 for 512 kB modules (some ESP-01, Sonoff Switch), 0xfc000 for 1 MB modules (most ESP-01), or 0x3fc000 for 4 MB modules (most ESP-12, Wemos D1).

**Comment by Italque** [comment]
Thanks for your help!
I had flashed the wrong sector with the "esp_init_data_default.bin", in my case. I have an ESP-12 but I had flashed the 0xfc000 sector although the 0x3fc000 was the right sector.
Now everything is solved!



Inviato da iPhone
Il giorno 07 gen 2018, alle ore 20:26, martin-ger <notifications@github.com<mailto:notifications@github.com>> ha scritto:


Did you try this:
Sometimes, especially on old ESP-01s, there is a wrong or non-matching version of "esp_init_data_default.bin" in the flash. If the firmware files from above flash correctly but after reboot you see only garbage on the serial and/or the LED on GPIO2 is flashing rapidly, try to re-initialize this sector: download https://github.com/espressif/ESP8266_NONOS_SDK/raw/master/bin/esp_init_data_default_v08.bin and flash it to 0x7c000 for 512 kB modules (some ESP-01, Sonoff Switch),  for 1 MB modules (most ESP-01), or  for 4 MB modules (most ESP-12, Wemos D1).
